user/how-to-guides/how-to-use-block-storage-devices.md

```diff
@@ -39,7 +39,7 @@ Click on one and your USB drive will be attached!
 However, it often means the app qube won't detect the new partition and you will need to manually mount it inside the app qube.
 See below for more detailed steps.
 
-## Block Devices in VMs
+## Block devices in VMs
```

Comment:
I don't understand why this word is being lowercased when the heading at the same level above it is in title case.